### PR TITLE
Feature/130897 more strings

### PR DESCRIPTION
**Pull request by cliffordp**
https://central.tri.be/issues/130897

Still have more strings in `src/admin-views/tribe-options-tickets.php` and then a final pass, but this is as far as I could get until later tonight.

### CODE REVIEW

**Comment by cliffordp** [comment]
@sc0ttkclark ready for your review.

# Do we want to add a new function for the `ticketed` word?
# There are texts in `.js` files, e.g. `'RSVP Capacity'` in `src/modules/blocks/rsvp/container-content/template.js` -- How should those be handled?

File: src/views/tickets/orders.php [hunk 1]
```diff
@@ -31,7 +31,7 @@
  * Display a notice if the user doesn't have tickets
  */
 if ( ! $view->has_ticket_attendees( $event_id, $user_id ) && ! $view->has_rsvp_attendees( $event_id, $user_id ) ) {
-	Tribe__Notices::set_notice( 'ticket-no-results', esc_html__( "You don't have tickets for this event", 'event-tickets' ) );
+	Tribe__Notices::set_notice( 'ticket-no-results', esc_html( sprintf( _x( "You don't have %s for this event", 'notice if user does not have tickets', 'event-tickets' ), tribe_get_ticket_label_plural_lowercase( 'notice_user_does_not_have_tickets' ) ) ) );
```
Comment (sc0ttkclark) [inline src/views/tickets/orders.php]: Needs version TBD change

File: src/views/tickets/email.php [hunk 2]
```diff
@@ -399,7 +399,7 @@
 													<span style="color:#0a0a0e !important; font-family: 'Helvetica Neue', Helvetica, sans-serif; font-size:15px;"><?php echo $ticket['ticket_id']; ?></span>
 												</td>
 												<td class="ticket-details" valign="top" align="left" width="120" style="padding: 0; width:120px; margin:0 !important;">
-													<h6 style="color:#909090 !important; margin:0 0 10px 0; font-family: 'Helvetica Neue', Helvetica, sans-serif; text-transform:uppercase; font-size:13px; font-weight:700 !important;"><?php esc_html_e( 'Ticket Type', 'event-tickets' ); ?></h6>
+													<h6 style="color:#909090 !important; margin:0 0 10px 0; font-family: 'Helvetica Neue', Helvetica, sans-serif; text-transform:uppercase; font-size:13px; font-weight:700 !important;"><?php echo esc_html( sprintf( _x( '%s Type', 'ticket type email heading', 'event-tickets' ), tribe_get_ticket_label_singular( 'ticket_type_email_heading' ) ) ); ?></h6>
```
Comment (sc0ttkclark) [inline src/views/tickets/email.php]: Needs version TBD change

File: src/views/blocks/rsvp/content-inactive.php [hunk 3]
```diff
@@ -11,16 +11,19 @@
...
  * @since 4.9
- * @version 4.9.4
+ * @since TBD Use function for text.
  *
+ * @version TBD
  */
 
-$message = $this->get( 'all_past' ) ? __( 'RSVPs are no longer available', 'event-tickets' ) : __( 'RSVPs are not yet available', 'event-tickets' );
+$message = $this->get( 'all_past' )
+	? sprintf( _x( '%s are no longer available', 'RSVP block inactive content', 'event-tickets' ), tribe_get_rsvp_label_plural( 'block_inactive_content' ) )
```
Comment (sc0ttkclark) [inline src/views/blocks/rsvp/content-inactive.php]:
```suggestion
	? sprintf( _x( '%s are no longer available', 'RSVP block inactive content in the past', 'event-tickets' ), tribe_get_rsvp_label_plural( 'block_inactive_content_past' ) )
```

**Comment by cliffordp** [comment]
@sc0ttkclark awaiting your response to my initial 2 questions

File: src/Tribe/Tickets.php [hunk 4]
```diff
@@ -1533,18 +1537,17 @@ protected function global_stock_mode_selector( $current_option = '' ) {
...
 		 * @return array
 		 */
 		protected function global_stock_mode_options() {
-			return array(
-				Tribe__Tickets__Global_Stock::GLOBAL_STOCK_MODE => __( 'Shared capacity with other tickets', 'event-tickets' ),
-				Tribe__Tickets__Global_Stock::OWN_STOCK_MODE    => __( 'Set capacity for this ticket only', 'event-tickets' ),
-			);
+			return [
+				Tribe__Tickets__Global_Stock::GLOBAL_STOCK_MODE => sprintf( _x( 'Shared capacity with other %s', 'global stock mode option', 'event-tickets' ), tribe_get_ticket_label_singular_lowercase( 'global_stock_mode_options' ) ),
+				Tribe__Tickets__Global_Stock::OWN_STOCK_MODE    => sprintf( _x( 'Set capacity for this %s only', 'global stock mode option', 'event-tickets' ), tribe_get_ticket_label_singular_lowercase( 'global_stock_mode_options' ) )
```
Comment (sc0ttkclark) [inline src/Tribe/Tickets.php]:
```suggestion
				Tribe__Tickets__Global_Stock::OWN_STOCK_MODE    => sprintf( _x( 'Set capacity for this %s only', 'individual stock mode option', 'event-tickets' ), tribe_get_ticket_label_singular_lowercase( 'global_stock_mode_options_individual' ) )
```

File: src/admin-views/editor/list-row.php [hunk 5]
```diff
@@ -40,7 +40,7 @@
 
 ?>
 <tr class="<?php echo esc_attr( $provider ); ?> is-expanded" data-ticket-order-id="order_<?php echo esc_attr( $ticket->ID ); ?>" data-ticket-type-id="<?php echo esc_attr( $ticket->ID ); ?>">
-	<td class="column-primary ticket_name <?php echo esc_attr( $provider ); ?>" data-label="<?php esc_html_e( 'Ticket Type:', 'event-tickets' ); ?>">
+	<td class="column-primary ticket_name <?php echo esc_attr( $provider ); ?>" data-label="<?php echo esc_html( sprintf( _x( '%s Type:', 'ticket type label', 'event-tickets' ), tribe_get_ticket_label_singular( 'ticket_type_label' ) ) ); ?>">
```
Comment (sc0ttkclark) [inline src/admin-views/editor/list-row.php]:
```suggestion
	<td class="column-primary ticket_name <?php echo esc_attr( $provider ); ?>" data-label="<?php echo esc_attr( sprintf( _x( '%s Type:', 'ticket type label', 'event-tickets' ), tribe_get_ticket_label_singular( 'ticket_type_label' ) ) ); ?>">
```

File: src/admin-views/editor/panel/ticket.php [hunk 6]
```diff
@@ -50,15 +50,15 @@ class="ticket_form_title tribe-dependent"
 					data-depends="#ticket_id"
 					data-condition-is-empty
 				>
-					<?php esc_html_e( 'Add new ticket', 'event-tickets' ); ?>
+					<?php echo esc_html( sprintf( _x( 'Add new %s', 'admin edit ticket panel heading', 'event-tickets' ), tribe_get_ticket_label_singular_lowercase( 'admin_edit_ticket_panel_heading' ) ) ); ?>
```
Comment (sc0ttkclark) [inline src/admin-views/editor/panel/ticket.php]:
```suggestion
					<?php echo esc_html( sprintf( _x( 'Add new %s', 'admin add new ticket panel heading', 'event-tickets' ), tribe_get_ticket_label_singular_lowercase( 'admin_add_new_ticket_panel_heading' ) ) ); ?>
```

File: src/admin-views/editor/panel/ticket.php [hunk 7]
```diff
@@ -72,15 +72,15 @@ class="ticket_form_title tribe-dependent"
 					data-depends="#ticket_id"
 					data-condition-is-empty
 				>
-					<?php esc_html_e( 'Add new RSVP', 'event-tickets' ); ?>
+					<?php echo esc_html( sprintf( _x( 'Add new %s', 'admin edit ticket panel heading', 'event-tickets' ), tribe_get_rsvp_label_singular( 'admin_edit_ticket_panel_heading' ) ) ); ?>
```
Comment (sc0ttkclark) [inline src/admin-views/editor/panel/ticket.php]:
```suggestion
					<?php echo esc_html( sprintf( _x( 'Add new %s', 'admin add new ticket panel heading', 'event-tickets' ), tribe_get_rsvp_label_singular( 'admin_add_new_ticket_panel_heading' ) ) ); ?>
```

File: src/admin-views/tribe-options-tickets.php [hunk 8]
```diff
@@ -200,10 +208,7 @@
 	? 'http://m.tri.be/19yk'
 	: 'http://m.tri.be/19yj';
 $paypal_setup_kb_link = '<a href="' . esc_url( $paypal_setup_kb_url ) . '" target="_blank">' . esc_html__( 'these instructions', 'event-tickets' ) . '</a>';
-$paypal_setup_note    = sprintf(
-	esc_html__( 'In order to use Tribe Commerce to sell tickets, you must configure your PayPal account to communicate with your WordPress site. If you need help getting set up, follow %s', 'event-tickets' ),
-	$paypal_setup_kb_link
-);
+$paypal_setup_note    = sprintf( esc_html_x( 'In order to use Tribe Commerce to sell %1$s, you must configure your PayPal account to communicate with your WordPress site. If you need help getting set up, follow %2$s', 'tickets fields settings PayPal setup', 'event-tickets' ), tribe_get_ticket_label_singular_lowercase( 'tickets_fields_settings_paypal_setup' ), $paypal_setup_kb_link );
```
Comment (sc0ttkclark) [inline src/admin-views/tribe-options-tickets.php]:
```suggestion
$paypal_setup_note    = sprintf( esc_html_x( 'In order to use Tribe Commerce to sell %1$s, you must configure your PayPal account to communicate with your WordPress site. If you need help getting set up, follow %2$s', 'tickets fields settings PayPal setup', 'event-tickets' ), esc_html( tribe_get_ticket_label_singular_lowercase( 'tickets_fields_settings_paypal_setup' ) ), $paypal_setup_kb_link );
```

**Comment by sc0ttkclark** [comment]
> Do we want to add a new function for the ticketed word?

Probably would be a good idea here, maybe consider a RSVP'd one too if we want to do that right.

> There are texts in .js files, e.g. 'RSVP Capacity' in src/modules/blocks/rsvp/container-content/template.js -- How should those be handled?

Let's do this and the above item in a future sprint though, please create a new ticket under triage to tackle this. I think we'd utilize the localized strings for the JS file part.

**Comment by sc0ttkclark** [comment]
Tests are failing due to an unrelated issue with `tribe_tickets_post_types` not having `tribe_events` in it. Will look into that.

**Pull request merged**